ngraph/core/src/op/parameter.cpp

```diff
@@ -48,6 +48,22 @@ void op::Parameter::set_is_relevant_to_shapes(bool is_relevant) {
     m_is_relevant_to_shapes = is_relevant;
 }
 
+bool op::Parameter::has_layout() const {
+    return get_output_tensor(0).get_rt_info().find("LAYOUT") != get_output_tensor(0).get_rt_info().end();
+}
+
+ov::Layout op::Parameter::get_layout() const {
+    auto it = get_output_tensor(0).get_rt_info().find("LAYOUT");
+    OPENVINO_ASSERT(it != get_output_tensor(0).get_rt_info().end(), "Layout for parameter is not set");
```

Comment:
what if we return `Layout("...")` if explicit layout is not set?
In this case we can write generic code and `"..."` == no information about layout, is not it?

Reply:
I don't see a problem with returning just Layout() (same as Layout(...)). In this case we probably not need 'has_layout' as user can check `param->get_layout() != Layout()` in the code

Reply:
yes, I agree